### ISSUE TITLE
AudioKit.start() crashes when an item is played in an AVPlayer

### ISSUE THREAD

**Issue by ydemartino** [AudioKit/AudioKit#1224]
I posted this question on stackoverflow because I'm experiencing a crash with AudioKit 4.0.4: https://stackoverflow.com/questions/48385938/fatal-error-audiokit-could-not-start-engine-when-calling-audiokit-start

The issue is that if am item is played in an AVPlayer, calling AudioKit.start() will crash the app. This works fine in a simulator.

I created a sample project that reproduces the bug: https://github.com/ydemartino/MicrophoneAnalysisTest

**Comment by aure** [comment]
If you're not using AudioKit for anything but mic tracking, use AKMicrophoneTracker which doesn't ever require an AudioKit signal chain.  Otherwise, you could maybe use AKPlayer and make that part of your signal chain, but I don't use AVPlayer and AudioKit isn't necessarily going to mix well with it.

**Comment by ydemartino** [comment]
I am using AudioKit only for mic tracking, but I also want to display the waveform with EZAudioPlot.
I tried to only use AKMicrophoneTracker but I'm not able to link it to an EZAudioPlot.

**Comment by aure** [comment]
Can you use AKPlayer?

**Comment by ydemartino** [comment]
Not really, I am using AVPlayer in a lot of screens, but on the specific screen I am using AudioKit, there is no AVPlayer instance. That's why I don't understand why it's crashing :/

**Comment by ydemartino** [comment]
I created the sample project from the microphone analysis example provided in AudioKit. My file is the exact copy of https://github.com/AudioKit/AudioKit/blob/master/Examples/iOS/MicrophoneAnalysis/MicrophoneAnalysis/ViewController.swift

The only thing I added was the InitialViewController which starts an AVPlayer and pauses the item before moving to the microphone analysis view controller.

This was just to demonstrate the issue. In my code, AudioKit is not controlled by a view controller lifecycle.
